filebeat/filebeat.template-es2x.json

```diff
@@ -11,14 +11,13 @@
       },
       "dynamic_templates": [
         {
-          "fields": {
```

Comment:
Yeah, it used to be the name of the dict/list field name. Now if everything is a keyword by default, the template also gets shorter, because we don't have dicts or lists of type `text`.